sql/src/test/java/io/crate/analyze/where/WhereClauseAnalyzerTest.java

```diff
@@ -253,248 +232,12 @@ public void testSelectWherePartitionedByColumn() throws Exception {
         AnalyzedUpdateStatement update = analyzeUpdate("update parted set id = 2 where date = 1395874800000");
         assertThat(update.query(), isFunction(EqOperator.NAME, isReference("date"), isLiteral(1395874800000L)));
     }
 
-    @Test
-    public void testClusteredByValueContainsComma() throws Exception {
-        WhereClause whereClause = analyzeSelectWhere("select * from bystring where name = 'a,b,c'");
-        assertThat(whereClause.clusteredBy(), contains(isLiteral("a,b,c")));
-        assertThat(whereClause.docKeys().get().size(), is(1));
-        assertThat(whereClause.docKeys().get().getOnlyKey(), isDocKey("a,b,c"));
-    }
-
-    @Test
-    public void testEmptyClusteredByValue() throws Exception {
-        WhereClause whereClause = analyzeSelectWhere("select * from bystring where name = ''");
-        assertThat(whereClause.clusteredBy(), contains(isLiteral("")));
-        assertThat(whereClause.docKeys().get().getOnlyKey(), isDocKey(""));
-    }
-
-    @Test
-    public void testClusteredBy() throws Exception {
-        WhereClause whereClause = analyzeSelectWhere("select name from users where id=1");
-        assertThat(whereClause.clusteredBy(), contains(isLiteral("1")));
-        assertThat(whereClause.docKeys().get().getOnlyKey(), isDocKey("1"));
-
-        whereClause = analyzeSelectWhere("select name from users where id=1 or id=2");
-
-        assertThat(whereClause.docKeys().get().size(), is(2));
-        assertThat(whereClause.docKeys().get(), containsInAnyOrder(isDocKey("1"), isDocKey("2")));
-
-        assertThat(whereClause.clusteredBy(), containsInAnyOrder(isLiteral("1"), isLiteral("2")));
-    }
-
-
-    @Test
-    public void testClusteredByOnly() throws Exception {
-        WhereClause whereClause = analyzeSelectWhere("select name from users_clustered_by_only where id=1");
-        assertFalse(whereClause.docKeys().isPresent());
-        assertThat(whereClause.clusteredBy(), contains(isLiteral(1L)));
-
-        whereClause = analyzeSelectWhere("select name from users_clustered_by_only where id=1 or id=2");
-        assertFalse(whereClause.docKeys().isPresent());
-        assertThat(whereClause.clusteredBy(), containsInAnyOrder(isLiteral(1L), isLiteral(2L)));
-
-        whereClause = analyzeSelectWhere("select name from users_clustered_by_only where id in (3,4,5)");
-        assertFalse(whereClause.docKeys().isPresent());
-        assertThat(whereClause.clusteredBy(), containsInAnyOrder(
-            isLiteral(3L), isLiteral(4L), isLiteral(5L)));
-
-
-        // TODO: optimize this case: there are two routing values here, which are currently not set
-        whereClause = analyzeSelectWhere("select name from users_clustered_by_only where id=1 and id=2");
-        assertFalse(whereClause.docKeys().isPresent());
-        assertTrue(whereClause.clusteredBy().isEmpty());
-    }
-
-    @Test
-    public void testCompositePrimaryKey() throws Exception {
```

Comment:
I think the cases here are covered in `EqualityExtractorTest` unless the routing values are relevant here for some reason?